speechbrain/inference/ASR.py

```diff
@@ -477,3 +482,390 @@ def transcribe_batch(self, wavs, wav_lens):
     def forward(self, wavs, wav_lens):
         """Runs full transcription - note: no gradients through decoding"""
         return self.transcribe_batch(wavs, wav_lens)
+
+
+@dataclass
+class ASRStreamingContext:
+    """Streaming metadata for a streaming transducer model. Initialized by
+    `TransducerASRStreamingWrapper.make_streaming_context`.
+
+    This object is intended to be mutated, so the same object should be passed
+    across calls as streaming progresses.
+
+    Holds some references to opaque streaming contexts, so the context is
+    model-agnostic to an extent."""
+
+    config: DynChunkTrainConfig
+    """Dynamic chunk training configuration used to initialize the streaming
+    context. Cannot be modified on the fly."""
+
+    fea_extractor_context: Any
+    """Opaque feature extractor streaming context."""
```

Comment:
This comment is not sufficient to understand what we are talking about. When reading the class, it's not clear to the user what should be passed here. Especially since there is no example of the class (maybe add an example of instanciation?)